astroplan/core.py

```diff
@@ -0,0 +1,234 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ..extern import six
+
+import sys
+from math import sqrt, pi, exp, log, floor
+from abc import ABCMeta, abstractmethod
+
+import numpy as np
+
+# Originally authored by Jazmin Berlanga Medina (jazmin.berlanga@gmail.com),
+# and modified by Adrian Price-Whelan (email) and Erik Tollerud (email).
+
+__all__ = ["Observer", "Date", "Target", "FixedTarget", "NonFixedTarget",
+           "Constraints", "Observation"]
+
+#__doctest_requires__ = {'*': ['scipy.integrate']}
+
+
+class Observer(object):
+    """
+    Some comments.
+    """
+    #def __init__(self, name, longitude, latitude, elevation, pressure,
```

Comment:
Ah, so this actually is almost what we want, but you don't want to comment out each function definition. Also, you want the `raise NotImplementedError()` below to be indented and under each function def, so, e.g.,

```python
def __init__(self, ...):
    """ 
    comments
    """
    raise NotImplementedError()

etc...
```

You should be able to run this file with `python astroplan/core.py` and it should just execute without any errors, but it won't do anything